libs/form-elements/src/Toggle.tsx

```diff
@@ -106,11 +113,11 @@ export default function Toggle({
     <div className={divClassName}>
       <Field {...props}>
         <div className={tokens.master}>
-          {reverse && label}
+          {reverse && label && <label className={labelClassName}>{label}</label>}
```

Comment:
is this an additional bugfix with missing label?

Reply:
the label already existed, I just added an HTML wrapper for styling purposes